tests/test_resources.py

```diff
@@ -49,5 +49,77 @@ def test_httpverb(self):
             response = method('foo')
             assert response.success
 
+    def test_pagination(self):
+      self.client.url = 'mock://api.com/{param}{?per_page}'
```

Comment:
hmm weird -- i just checked and you're right. I was following the [documentation](https://developer.github.com/guides/traversing-with-pagination/#changing-the-number-of-items-received)